auto_nag/scripts/crash_small_volume.py

```diff
@@ -0,0 +1,219 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag.history import History
+from auto_nag.topcrash import TOP_CRASH_IDENTIFICATION_CRITERIA, Topcrash
+
+# TODO: should be moved when resolving https://github.com/mozilla/relman-auto-nag/issues/1384
```

Comment:
We should finally fix this, this TODO is everywhere now :rofl:

Reply:
I will prioritize it :)